api/helpers/redis/subscribe.js

```diff
@@ -0,0 +1,24 @@
+const redis = require('redis');
+
+module.exports = {
+  friendlyName: 'Subscribe',
+  inputs: {
+    channel: {
+      required: true,
+      type: 'string'
+    },
+  },
+
+  exits: {
+
+  },
+
+  fn: async function (inputs, exits) {
+
+    const subscriber = redis.createClient({ url: process.env.REDISSTRING });
```

Comment:
as we learned tonight, this is going to create a connection for each message queue, which is done by server.
so lets say 1 zombie kill, 1 playerkill, * 500 servers = > 1000 connections. Plus publishing. It starts to die fast.

I think this needs to be setup like the bull queues. General queues that have server info on them.

Reply:
I actually can't think of a reason why we shouldn't just use bull queues instead of redis here.